custom-recipes/meaningcloud-sentiment-analysis/recipe.json

```
@@ -1,14 +1,12 @@
-// This file is the descriptor for the Custom code recipe meaningcloud-sentiment-analysis
 {
     "meta": {
         "label": "Sentiment analysis",
```

```suggestion
        "label": "Sentiment Analysis",
```
The labels of all the other requests are capitalized.

---

Good catch!